src/deploy/functions/runtimes/supported/types.ts

```diff
@@ -75,37 +75,43 @@ export const RUNTIMES = runtimes({
     friendly: "Node.js 18",
     status: "GA",
     deprecationDate: "2025-04-30",
```

Comment:
While you're editing this file can yo ualso update Node16 and below to be "decommissioned" for their status?